pyramda/iterable/reject.py

```diff
@@ -0,0 +1,14 @@
+from pyramda.function.curry import curry
+from . import filter
+
+
+@curry
+def reject(f, xs):
```

Comment:
For predicates (functions returning booleans) prefer the variable name `p`. The name `f` implies it could be an arbitrary function, which isn't the case.